tests/e2e/features/createRecipe.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Create recipe feature
+    As a user
+    I want to create a recipe
+    So that I can store and share with other users
+
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | user     |
+            | password         | aaaa1234 |
+            | confirm_password | aaaa1234 |
```

Comment:
- confirm_password can be omitted
- format data table
- use some good user names
```suggestion
        Given the user has signed up with the following details
            | username | Alice |
            | password | 1234  |
```

Reply:
confirm password is used just to reuse step from previous scenario rather than making new one
or should i ?

Reply:
Then the step should be fine for now. But please, push a refactor PR for that.

Reply:
That should be implemented in https://github.com/KarunAtreya/recipes/pull/14